setup.cfg

```diff
@@ -13,6 +13,9 @@ classifiers =
 	Operating System :: POSIX :: Linux
 	Programming Language :: Python
 	Programming Language :: Python :: 3
+	Programming Language :: Python :: 3.12
+	Programming Language :: Python :: 3.11
+	Programming Language :: Python :: 3.10
```

Comment:
- [x] Version bump ⬆️ 
🤔 These are the supported versions anyway. Should consider even removing older versions in next cycle.